test/performance/odp_pktio_perf.c

```diff
@@ -52,6 +52,12 @@
  * received by this time will be assumed to have been lost. */
 #define SHUTDOWN_DELAY_NS (ODP_TIME_MSEC_IN_NS * 100)
 
+/* Number of duration units in a second. */
+#define T_SCALE 10
```

Comment:
How about milliseconds?

Reply:
I am not sure how that would be better other than milliseconds being more typical a unit than the tenths of seconds used now. One benefit of not using finer time scale than necessary is that duration can be longer before it overflows.

Reply:
> more typical a unit

That is definitely reason enough for me. If 20+ days is not a sufficient duration, then make it 64 bit. But I leave it to you to decide.